tagent/src/handlers.rs

```diff
@@ -105,7 +100,7 @@ pub fn get_local_listing(full_path: PathBuf) -> Vec<String> {
     result
 }
 
-type FileListHttpRsp = Either<HttpResponse, web::Json<FileListingRsp>>;
+type FileListHttpRsp = Result<web::Json<FileListingRsp>, TagentError>;
```

Comment:
Love it! Not a big fan of those `Either` that were there before, to be honest. But the explicit error makes it so much better now.